README.md

```diff
@@ -343,5 +352,33 @@ The generated report will be available at ```target/site/jacoco/index.html```
+| `customHttpHeader`                         | Add a [custom HTTP header](#custom-http-headers) to the list of HTTP headers that will the client sends to the Unleash API. Each method call will add a new header. | No       | N/A                                  |
+| `customHttpHeadersProvider`                | Add a custom HTTP header provider. Useful for [dynamic custom HTTP headers](#dynamic-custom-http-headers).                                                          | No       | `null`                               |
+| `disableMetrics`                           | A boolean indicating whether the client should disable sending usage metrics to the Unleash server.                                                                 | No       | `false`                              |
+| `enableProxyAuthenticationByJvmProperties` | Enable support for [using JVM properties for HTTP proxy authentication](#http-proxy-with-authentication).                                                           | No       | `false`                              |
+| `environment`                              | The name of the current environment.                                                                                                                                | No       | `null`                               |
+| `fallbackStrategy`                         | ???                                                                                                                                                                 | No       | `null`                               |
+| `fetchTogglesInterval`                     | How often (in seconds) the client should check for toggle updates.                                                                                                  | No       | `10`                                 |
+| `instanceId`                               | An identifier for your instance. Typically a hostname, pod id or something similar. Unleash uses this internally for ...something                                   | Yes      | `null`                               |
+| `namePrefix`                               | ???                                                                                                                                                                 | No       |                                      |
+| `projectName`                              | The name of the Unleash project?                                                                                                                                    | No       |                                      |
+| `proxy`                                    | ???                                                                                                                                                                 | No       |                                      |
```

Comment:
Is this the HTTP proxy that's referenced other places in the readme? What is that?

Reply:
This is an object containing proxy details in case there's a third party proxy between your SDK and your Unleash server